docs/src/userguide/loading_iris_cubes.rst

```diff
@@ -215,6 +215,14 @@ constraint to ``load``::
     level_10 = iris.Constraint(model_level_number=10)
     cubes = iris.load(filename, forecast_6 & level_10)
 
+.. note::
+
+    Whilst ``&`` is supported, the ``|`` that might reasonably be expected
+    is not. This is because "or" constraints could lead to a cube which did
+    not cover a hyper-rectangular region if these constraints were on
```

Comment:
"hyper-rectangular" is perhaps a tad technical, consider just "rectangular" or "boxlike".

Reply:
Done, good suggestion, thanks!